winlogbeat/sys/strings_windows.go

```diff
@@ -0,0 +1,45 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package sys
+
+import (
+	"golang.org/x/sys/windows"
+	"golang.org/x/text/encoding"
+	"golang.org/x/text/encoding/charmap"
+)
+
+var ansiDecoder *encoding.Decoder
+
+func init() {
+	ansiCP := windows.GetACP()
```

Comment:
Not 100% about this one. Not sure if would be better to just add a config option that defaults either to ACP or Windows1250 and that can be user defined. I am not super confident the active page will always be the one we want.

Reply:
or rather option to use specific ANSI code page or let us discover it if not set
However I think the `GetACP` will return the ANSI code page we want, because if people are using non-English variants of OS the winlog messages are also non-English